prepare_nutrients.py

```diff
@@ -65,6 +75,13 @@ def assign_weightings(product):
         except Exception as e:
             computed_nutrient['notes'] = e
 
+        if nutrient_key == 'salt':
+            computed_nutrient['weighting'] = 100
+            continue
+         
+
+        computed_nutrient['weighting'] = 1
```

Comment:
Is this a permanent change or just an experiment? Breaks a unit test (I have commented the test for now).

Reply:
I tried many values and looked at the resulting accuracy: https://docs.google.com/document/d/1NFg2lAaDzSkRldKU6uDGcjPVxbfTsc9p59r7L6OEqiQ/edit#heading=h.fl6o8g6m5uv5

So far it's the best I found.